frontend/src/libraries/explorer-wamp/nodes.ts

```diff
@@ -23,10 +24,10 @@ export default class NodesApi extends ExplorerApi {
         `SELECT ip_address as ipAddress, moniker, account_id as accountId, node_id as nodeId, signature, 
                 last_seen as lastSeen, last_height as lastHeight, last_hash as lastHash,
                 agent_name as agentName, agent_version as agentVersion, agent_build as agentBuild,
-                peer_count as peerCount, is_validator as isValidator
+                peer_count as peerCount, is_validator as isValidator, status
                     FROM nodes
                     ${endTimestamp ? `WHERE last_seen < :endTimestamp` : ""}
-                    ORDER BY last_seen DESC
+                    ORDER BY is_validator, last_seen DESC
```

Comment:
I don't think we need this as this will bring old validator nodes up.

Reply:
I change the sequence since we still want sort by isValidator.
`ORDER BY last_seen, is_validator DESC`

Reply:
Based on the Illia's request, we should have just two tabs: `WHERE is_validator = 1` and `WHERE is_validator = 0` + `AND last_seen < 60 seconds ago` (hide offline / non-active nodes)